lib/dal/src/management/prototype.rs

```diff
@@ -589,6 +571,43 @@ impl ManagementPrototype {
         })
     }
 
+    async fn variant_socket_map(
+        ctx: &DalContext,
+    ) -> ManagementPrototypeResult<HashMap<String, u32>> {
+        // Count sockets on uninstalled schemas
+        let mut variant_socket_map: HashMap<_, _> = CachedModule::latest_modules(ctx)
```

Comment:
This is where we add in the uninstalled schemas to the variantSocketMap. Instead of checking which schemas are installed, we just get them all and then overwrite the installed ones in the next loop.